_includes/newcomers/newcomers.html

```diff
@@ -0,0 +1,99 @@
+</p>
+<table style="margin:0px; padding:0px;width:100%; table-layout: fixed;">
+    <th>Information</th>
+    <th>Guides</th>
+    <th>Issues up for grabs</th>
+    <tr>
+        <td><a href="https://bit.ly/2SbrRhe" target="_blank">Calender</a> | <a href="http://slack.layer5.io"
+                target="_blank">Slack</a> | <a href="https://github.com/topics/meshery" target="_blank">GitHub</a> | <a
+                href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1"
+                target="_blank">YouTube</a> | <a href="https://drive.google.com/drive/folders/0ABH8aabN4WAKUk9PVA"
+                target="_blank">Drive</a></td>
+        <td><a href="https://docs.google.com/document/d/1tpg2sLxirozNt3Ofr3GdM002f9rExp74EqrsGZBU710/edit?usp=sharing"
+                target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+"
+                target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit"
+                target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3Ahacktoberfest"
+                target="_blank">Hacktoberfest</a></td>
+    </tr>
+    <tr>
+        <td><a href="https://meet.layer5.io/newcomers" target="_blank">Weekly Zoom on Thursdays at 8AM Central</a></td>
+        <td><a href="https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.504assxl7gpf"
+                target="_blank">Layer5 Repository Overview</a></td>
+        <td><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22"
+                target="_blank">good first issue</a></td>
+    </tr>
+</table>
+
+<h3>How to contribute?</h3>
+
+<p>
+    The following set of steps outlines the process by which you can openly engage, learn, and participate in the broad
+    set of open source projects at Layer5. If at any time you get stuck, please seek help in the #newcomers channel
+    in the Layer5 (joining instructions below). Our <a target="_blank"
+        href="https://layer5.io/community/meshmates">MeshMates</a> and community members are here to help!
+</p>
+
+<h4>Finding an issue to work on</h4>
+
+<ol>
+    <li>Browse the list of <a href="https://github.com/layer5io" target="_blank">Layer5 org</a> issues bearing the “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22+"
+            target="_blank">good first issue</a>” label, “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22first+timers+only%22+"
```

Comment:
`https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22first-timers-only%22+` is the proper tag to use here